docs/source/get_involved/governance.rst

```diff
@@ -553,6 +593,20 @@ the CC tries to resolve the deadlock.
 -  Any proposal reaching stage 3 must be supported by an :ref:`steps`,
    which has been made public at least 5 days, excluding weekends, before the vote.
 
+Judication
+~~~~~~~~~~
+
+Besides its primary remit, the code of conduct, the CoC committee is empowered to
+judicate in contentious cases of application of the following rules by the CC:
+
+* voting rules, vote counts, start/end times
+* determining activity/inactivity of a core developer
+* determining unresponsiveness of an algorithm maintainer
+* inclusion and exclusion criteria for algorithms
+
+Judication is upon written appeal of core developers and algorithm maintainers to the CoC committee.
+A ruling by the CoC committee does not halt timelines, but can reverse decisions made.
+
```

Comment:
I would propose to remove that part, I think thats not what CoCC is meant to do, CoCC is meant to make sure that CoC is applied. But this here is adding additional power, e.g. judging a core developers activities. Can we kindly remove that section?